api/buildout.cfg

```diff
@@ -1,19 +1,26 @@
 [buildout]
 index = https://pypi.org/simple/
-extends = http://dist.plone.org/release/5.2.6/versions.cfg
-
-parts = instance plonesite site-packages robot-server
+extends =
+    http://dist.plone.org/release/5.2.6/versions.cfg
+    version-constraints.cfg
+    versions.cfg
+parts = instance plonesite site-packages test robot-server
+show-picked-versions = true
+update-versions-file = versions.cfg
 versions = versions
 
 extensions = mr.developer
 auto-checkout =
+    plone.rest
+    plone.restapi
+    plone.volto
 always-checkout = force
 show-picked-versions = true
 
 [sources]
-plone.volto = git https://github.com/plone/plone.volto.git branch=main
-plone.rest = git git@github.com:plone/plone.rest.git branch=master
-plone.restapi = git git://github.com/plone/plone.restapi.git pushurl=git@github.com:plone/plone.restapi.git branch=master
+plone.volto = git https://github.com/plone/plone.volto.git branch=test-fix-frontend-domain-none
```

Comment:
I think this changes shouldn't be committed here. You can add them as instructions in the PR comments, but with them here it makes it hard to merge. I understand they're prerequisite for the PR.

Reply:
Done.